ui/src/app/components/runtime-configuration-panel/disk-selector.tsx

```diff
@@ -37,6 +67,26 @@ export const DiskSelector = ({
     <FlexColumn
       style={{ ...styles.controlSection, gap: '11px', marginTop: '11px' }}
     >
+      {!disableDetachableReason && (
+        <FlexRow style={warningBoxStyle.box}>
+          <WarningIcon style={warningBoxStyle.icon} />
+          <div>
+            <AoU /> will now only support re-attachable persistent disks as the
+            storage disk option and will discontinue standard disks. Refer to
+            the
+            <a
+              target='_blank'
+              href={
+                'https://support.researchallofus.org/hc/en-us/articles/360041683891'
```

Comment:
Is this the right link?  I see an article "How do I use Workspace Buckets to save my files in notebooks"